nexus/lib/qmcpack_converters.py

```diff
@@ -864,6 +864,19 @@ def check_sim_status(self):
         output = open(os.path.join(self.locdir,self.outfile),'r').read()
         #errors = open(os.path.join(self.locdir,self.errfile),'r').read()
 
+        # Recent versions of convert4qmc no longer produce the orbs.h5 file.
```

Comment:
Do you know how recent? If not, put 3.15 should be fine.

Reply:
It's not very recent.  @anbenali knows better.

Reply:
convert4qmc produces an HDF5 only when a txt file is provided (gamess) and the tag -hdf5 is produced.